### PR TITLE
Document number_format() $decimals change

### PR DESCRIPTION
**Pull request by iluuu1994**
https://github.com/php/php-src/issues/13866

### CODE REVIEW

File: appendices/migration83/incompatible.xml [hunk 1]
```diff
@@ -196,6 +196,11 @@
     <parameter>$step</parameter> is a float when trying to generate a range of
     characters, except if both boundary inputs are numeric strings (e.g.
     <code>range('5', '9', 0.5);</code> does not produce a warning).</member>
+    <member><function>number_format()</function> now handles negative
+    <parameter>$decimals</parameter> values by rounding
+    <parameter>$num</parameter> to <code>abs($decimals)</code> digits before the
+    decimal point.  Prevoiusly, negative <parameter>$decimals</parameter> values
```
Comment (devnexen) [inline appendices/migration83/incompatible.xml:202]: typo `Prevoiusly` + needs closing `</member>` I think.

Comment (iluuu1994) [reply appendices/migration83/incompatible.xml]: Thanks. I don't have a build setup for docs.  :smile:

**Review by devnexen — APPROVED** [review]

**Comment by Girgias** [comment]
This probably needs an entry in https://github.com/php/doc-en/issues/2796 and update the `number_format` docs too?

**Comment by iluuu1994** [comment]
@Girgias Seems it's already there (The $decimal of number_format() now properly handles negative integers.)

**Comment by Girgias** [comment]
Oh indeed, so just need to amend the function doc page then. :)

**Pull request merged**